.github/workflows/node.js.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x, 15.x]
```

Comment:
what's the benefit of building & testing on all these different node versions?  I would think all developers should stick to a defined version.

Reply:
I _think_ this helps in ensuring that the project works on different node versions
But I agree with you, this can also lead to weird ci-fails, where it fails since we use node 14/15 and something might be different compared to node 11

Reply:
yeah, I forsee random issues popping up with stuff failing on node 10.x even though we are all using node 15.x.  Just a time sink imo.

Reply:
True, I used a random Github action template, That can be changed to just 14 and 15